### DISCUSSION TITLE
How to sort by distance?

### DISCUSSION THREAD

**Discussion by iamvaliyev** [prisma/prisma#5701] [Q&A]
I did not find in the manual how to sort data by geolocation. How can i do this ?

**Comment by ryands17** [comment]
@iamvaliyev 👋 

Currently we do not have [native support](https://www.prisma.io/docs/concepts/components/preview-features/native-types/native-types-mappings#unsupported) for spatial data types, but you can find distances via a raw query as shown in [this](https://github.com/prisma/prisma-examples/tree/latest/typescript/postgis-express) example.

**Reply by amitkadivar** [reply]
any update on this? 
I have a similar case like i get the latitude and longitude from the user and in db I have some properties so I need to sort those properties distance-wise. i also use other filters as well so if any why then please guide me

**Reply by preAGIcoder** [reply]
@amitksigma did you figure it out?  I am facing similar issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
